package.json

```diff
@@ -19,8 +19,8 @@
   },
   "lint-staged": {
     "packages/*/src/**/*.{ts,tsx}": [
-      "tslint -c tslint.json --fix",
       "prettier --write",
+      "tslint -c tslint.json --fix",
```

Comment:
I changed the order here since prettier fixes the tslint line length rule that has no fixer.